docker/dev/php/Dockerfile

```diff
@@ -0,0 +1,46 @@
+FROM php:7.4
```

Comment:
@elacunza weren't we targetting php 7.3 ?
If so we'd better stick with 7.3 also in development images so that we don't use php 7.4's syntactic sugar inadvertendly.